### DISCUSSION TITLE
Question about the comma in palettes setting.

### DISCUSSION THREAD

**Discussion by himcraft** [JuliaPlots/Makie.jl#2816] [Q&A] [answered]
Hi all,

Sorry for the dumb question, but when I tried to reproduce the results in the [document section about palettes](https://docs.makie.org/stable/documentation/theming/#palettes), I noticed that when I typed without the last comma as:
```julia
Axis(f[2, 1],
    title = "Custom cycle palette",
    palette = (patchcolor = [:red, :green, :blue, :yellow, :orange, :pink]))
```
instead of the original code given in the doc:
```julia
Axis(f[2, 1],
    title = "Custom cycle palette",
    palette = (patchcolor = [:red, :green, :blue, :yellow, :orange, :pink],))
```

It threw an error `MethodError: no method matching iterate(::Symbol)`. 

Can someone please explain to me the usage of the last comma?  Thanks a lot!

**Comment by SimonDanisch** [comment, marked as answer]
That's because `(x=2)` is not `(; x=2)` or `(x=2,)` ;) The latter create a named tuple, while the first one just wraps it in brackets ;)

**Reply by himcraft** [reply]
I got. Thank you for the example. :)
